Forecasts/Moduls/City/CityInteractor.swift

```diff
@@ -67,9 +75,8 @@ class CityInteractor: CityInteractorProtocol {
             }
         }
         if (isExist == false) {
-            self.citys.append(CityModel(city: citySearch, tempr: "", link: ""))
+            self.citys.append(CityModel(city: citySearch, tempr: "", link: "", icon: ""))
```

Comment:
You could change CityModel init by adding default arguments.

1. Add default value for tempr, link, icon
var tempr: String = “”
....
or
2. Add custom init with default arguments
init(city: String, tempr: String = “”, link: String,...

It will make this line briefer.
self.cities.append(CityModel(city: citySearch))